modules/build-buildscript.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
-	classpath group: "com.liferay", name: "com.liferay.gradle.plugins.defaults", version: "5.4.99"
+	classpath group: "com.liferay", name: "com.liferay.gradle.plugins.defaults", version: "5.4.100"
```

Comment:
If you're making changes to a Gradle plugin, please update its `CHANGELOG.markdown` (and its `README.markdown` if it exists).